addons/supplies/config.cpp

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {
+            "tac_supply_Launchers",
+            "tac_Supply_Engineer",
+            "tac_Supply_Medic"
+        };
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"tac_main", "cba_main", "ace_medical_treatment", "AMP_Breaching_Charge"};
```

Comment:
`"AMP_Breaching_Charge"`
Excuse me?

Must go into an optional, no dependencies on smaller mods, makes it harder to remove, maintain and test.